packages/venia-ui/lib/components/Routes/routes.js

```diff
@@ -1,34 +1,17 @@
-import React, { Suspense, lazy } from 'react';
+import React, { Suspense } from 'react';
 import { Route, Switch, useLocation } from 'react-router-dom';
 
 import { fullPageLoadingIndicator } from '../LoadingIndicator';
 import MagentoRoute from '../MagentoRoute';
 import { useScrollTopOnChange } from '@magento/peregrine/lib/hooks/useScrollTopOnChange';
 
-const CartPage = lazy(() => import('../CartPage'));
-const CheckoutPage = lazy(() => import('../CheckoutPage'));
-const CreateAccountPage = lazy(() => import('../CreateAccountPage'));
-const Search = lazy(() => import('../../RootComponents/Search'));
```

Comment:
Inspecting targeted files such as `routes` results in a clear difference between the source (which has the injected routes) and what is displayed by the browser (prettified but doesn't show any injected routes). I'm not sure how much work it will be to modify the source map generation to process the file POST build/intercept but I would like to at least talk about it. Currently we just have the routes and rich content renderer being intercepted but if we start making a habit of using extensibility the developer debug experience will get rough.

Reply:
The talon interceptors work with source maps because they modify the source code text before Babel parses it and creates a source map. I agree that we need to regenerate source maps when we make changes like this to module code, but it was pretty tricky to figure out in the Webpack plugin. Would you be comfortable making it a followup story?

Reply:
A follow up would be fine since this is the only non-talon intercept.